airflow/jobs/job.py

```diff
@@ -309,8 +398,8 @@ def execute_job(job: Job | JobPydantic, execute_callable: Callable[[], int | Non
     database operations or over the Internal API call.
 
     :param job: Job to execute - it can be either DB job or it's Pydantic serialized version. It does
-       not really matter, because except of running the heartbeat and state setting,
-       the runner should not modify the job state.
+    not really matter, because except of running the heartbeat and state setting,
+    the runner should not modify the job state.
```

Comment:
Why change this? Doesn’t really matter but the indentation is good for readability.

Reply:
No sure, probably some formatting tool. Reverted